workloads/files/workload-http-script-cm.yml

```diff
@@ -0,0 +1,1217 @@
+      local completed
+      while true; do
+        completed=$(oc get pods --no-headers -n=$centos_stress_ns -l=app=centos-stress --field-selector=status.phase=Succeeded 2>/dev/null | wc -l)
+        echo "Completed wlg pods: $completed/${LOAD_GENERATORS}"
+        test $completed -eq ${LOAD_GENERATORS} && break
+        sleep 10
+      done
+    }
+
+    oc_cleanup
+    oc_create
+    client_wait_complete
+    oc_cleanup
+
+  ###################################################################################################
+  stress-pod.yaml: |
+    apiVersion: v1
+    kind: Template
+    metadata:
+      name: centos-stress-template
+    objects:
+    - apiVersion: v1
+      kind: Pod
+      metadata:
+        generateName: centos-stress-
+        labels:
+          app: centos-stress
+      spec:
+        affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - centos-stress
+              topologyKey: kubernetes.io/hostname
+        containers:
+        - env:
+          - name: IDENTIFIER
+            value: ${IDENTIFIER}
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          envFrom:
+          - configMapRef:
+              name: wlg-env
+          image: quay.io/jmencak/centos-stress
+          imagePullPolicy: Always
+          name: centos-stress
+          resources: {}
+          volumeMounts:
+          - mountPath: /etc/kubernetes/
+            name: k8s-config
+          - mountPath: /opt/wlg/
+            name: targets
+          - mountPath: /opt/stress/.ssh/
+            name: ssh-key
+        hostNetwork: false
+        nodeSelector: "${{NODE_SELECTOR}}"
+        restartPolicy: Never
+        securityContext:
+          sysctls:
+          - name: net.ipv4.ip_local_port_range
+            value: "1024 65535"
+    # "unsafe" sysctl
+    #      - name: net.ipv4.tcp_tw_reuse
+    #        value: "1"
+    #    serviceAccountName: stress
+        tolerations:
+        - effect: NoSchedule
+          key: test
+          operator: Equal
+          value: wlg
+        volumes:
+        - configMap:
+            items:
+            - key: k8s-config
+              path: k8s.conf
+            name: k8s-config
+            optional: true
+          name: k8s-config
+        - configMap:
+            items:
+            - key: wlg-targets
+              path: targets.txt
+            name: wlg-targets
+            optional: true
+          name: targets
+        - name: ssh-key
+          secret:
+            items:
+            - key: wlg-ssh-key
+              path: id_rsa
+            optional: true
+            secretName: wlg-ssh-key
+            # yaml/json doesn't support octal: 400 = (4 * 8^2) + (4 * 8^1) + (4 * 8^0) => 256
```

Comment:
I am not sure I agree with this comment - https://github.com/openshift-scale/workloads/blob/master/workloads/templates/workload-job.yml.j2#L86

however perhaps the context in how the reference above is rendered (j2 -> yaml), might be providing the value for defaultMode in the correct "type"

Reply:
Interesting, didn't work for me when I tested, but will double check and change to `0400` if that really works.